### ISSUE TITLE
Possibility for a 0.7.4 release?

### ISSUE THREAD

**Issue by pjonsson** [JoshuaKGoldberg/TypeStat#2012]
### 🚀 Feature Request

I ran `npx typestat` which asked to install 0.7.3, and then I quickly moved on to hitting what looks like #1427 both when trying a js-migration and a ts-improvement.

#### Existing Behavior

```
Error in /files/lib/Core/Json.ts at node 'export interface JsonObject {
  [key: string]: JsonValue;
}' (position 103):
        TypeError: Cannot read properties of undefined (reading '0')
            at expandReferencesForGenericTypes (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typestat/src/mutators/builtIn/fixIncompleteTypes/fixIncompleteInterfaceOrTypeLiteralGenerics/collectGenericNodeReferences.js:83:77)
            at collectGenericNodeReferences (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typestat/src/mutators/builtIn/fixIncompleteTypes/fixIncompleteInterfaceOrTypeLiteralGenerics/collectGenericNodeReferences.js:39:56)
            at visitInterfaceOrTypeLiteral (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typestat/src/mutators/builtIn/fixIncompleteTypes/fixIncompleteInterfaceOrTypeLiteralGenerics/index.js:39:99)
            at tryGetMutation (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typestat/src/mutators/collectMutationsFromNodes.js:50:16)
            at visitNode (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typestat/src/mutators/collectMutationsFromNodes.js:37:30)
            at visitNodes (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typescript/lib/typescript.js:32142:22)
            at forEachChildInSourceFile (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typescript/lib/typescript.js:32355:12)
            at Object.forEachChild (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typescript/lib/typescript.js:32655:35)
            at collectMutationsFromNodes (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typestat/src/mutators/collectMutationsFromNodes.js:44:8)
            at fixIncompleteInterfaceOrTypeLiteralGenerics (/home/node/.npm/_npx/42ab35185b79194c/node_modules/typestat/src/mutators/builtIn/fixIncompleteTypes/fixIncompleteInterfaceOrTypeLiteralGenerics/index.js:34:124)
```

#### Change Proposal

Since the bug seems to have been fixed on main branch for quite a few months, it would be nice with a 0.7.4 release that contains the fix.

**Comment by JoshuaKGoldberg** [comment]
O, thank you for saying something! There was _supposed_ to be an automated release. But it looks like the release flow was failing. I fixed it up just now and we've got a `0.8.0` release: https://www.npmjs.com/package/typestat/v/0.8.0

Amusingly, there's no GitHub Release associated, because the total number of pent-up changes went past the release body limit:

```plaintext
A  CHANGELOG.md
 M package.json
ERROR 422 (Validation Failed: {"resource":"Release","code":"custom","field":"body","message":"body is too long (maximum is 125000 characters)"} - https://docs.github.com/rest/releases/releases#create-a-release)
```

**Comment by pjonsson** [comment]
@JoshuaKGoldberg got a 0.8.1 release through npx now, and that doesn't immediately give the backtrace, thank you!
